Assets/Scripts/SpawnController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+
+public class SpawnController : MonoBehaviour {
+
+	private float _scaleFactor;		// What to scale each 1x1 unit by to fit on the screen
+
+
+#region Setters and Getters
+	// Returns the scale factor for each sprite
+	public float ScaleFactor {
+		get{return _scaleFactor;}
+		set{_scaleFactor = value;}
+	}
+#endregion
+
+
+	// Call this to instantiate a new game grid
+	public Token[][] CreateGrid(int size) {
+		// Initialize Tokens jagged array
+		Token[][] tokens = new Token[size][];
```

Comment:
Not necessarily an issue with the existing code, but this limits us to only square maps.  Possibly take width and height to allow for more variance.